pandas/tests/frame/test_reductions.py

```diff
@@ -1763,3 +1763,85 @@ def test_prod_sum_min_count_mixed_object():
     msg = re.escape("unsupported operand type(s) for +: 'int' and 'str'")
     with pytest.raises(TypeError, match=msg):
         df.sum(axis=0, min_count=1, numeric_only=False)
+
+
+def test_timezone_min_max_with_nat():
+    # GH#27794
+    df = pd.concat(
+        [
+            pd.date_range(start="2018-01-01", end="2018-01-03")
+            .to_series()
+            .dt.tz_localize("UTC"),
+            pd.date_range(start="2018-01-01", end="2018-01-02")
+            .to_series()
+            .dt.tz_localize("UTC"),
+        ],
+        axis=1,
+    )
```

Comment:
we try to avoid using concat in tests that aren't aimed at concat.  simpler to write this as

```
df = DataFrame({
    "A": pd.date_range(start="2018-01-01", end="2018-01-03", tz="UTC"),
    "B": pd.date_range(start="2018-01-01", end="2018-01-02", tz="UTC")
})
```

Reply:
thanks, done.